support-frontend/assets/helpers/user/getUser.js

```diff
@@ -1,13 +1,13 @@
 // @flow
 import { type Option } from 'helpers/types/option';
 
-export type User = {|
+export type GetUser = {|
```

Comment:
Can we keep this type as `User` please as that is what it represents.

Reply:
yes of course, another intellij mistake there that I hadn't self reviewed yet